### ISSUE TITLE
Feature request: support for winston

### ISSUE THREAD

**Issue by jessekelly881** [sukovanej/effect-log#18]
[Winston](https://www.npmjs.com/package/winston) would be a useful addition to this set of loggers. Especially considering its support for and ecosystem of remote loggers like files, dbs, third party services like logtail,, etc. I would be willing to create a PR, but I'm wondering if there is interest in adding support for this to this lib or whether it might be better to make this a separate lib(considering deps)

**Comment by jessekelly881** [comment]
Actually, having explored this more, this should definitely be a separate library. I don't think it makes too much sense here. Closing.

**Comment by sukovanej** [comment]
I don't really have a clear vision for this library. I created it solely to cover my use-cases for stdout local and non-local logging. At this point, I'd rather vote for a separate lib, honestly. 

I use winston as well. I guess it's a pretty popular logging solution in general, thus it might be worth checking the effect-ts discord and maybe somebody already prepared a solution. If not, I'd be willing to convert this codebase into a monorepo and expose a package to simplify winston integration from here. Unfortunately, I can't really promise it'll happen in a near future.

**Comment by jessekelly881** [comment]
Makes sense! I'll ask around. The logger I linked above satisfies my use case and I will probably just use that for the time being. Also, I wonder a full port of winston to effect makes sense.. instead of relying on winston to handle the different remote connections.
